path-traversal/extract_files.py

```
@@ -0,0 +1,83 @@
+    repo_safe = repo.replace('/', '_')
+    output_file = os.path.join(output_dir, f"{repo_safe}_{safe_filename}")
+    
+    try:
+        response = requests.get(raw_url, timeout=30)
+        response.raise_for_status()
+        
+        with open(output_file, 'w', encoding='utf-8') as f:
+            f.write(response.text)
+        
+        print(f"Downloaded: {output_file}")
+        return True
```

Ruff (0.13.1)

Consider moving this statement to an `else` block

(TRY300)